app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -32,65 +32,72 @@ object FullResultsPageHelper extends ViewHelper {
   private val govukTable = new GovukTable()
   private val govukDetails = new GovukDetails()
 
-  def displayFullCalculationResult(
-    calculatorResult: CalculatorResult,
+  def nonTabCalculationResultsTable(
```

Comment:
If we decided to go on separate pages for each year, this may have to be split into multiple methods. Easiest thing to do is to return an array of Htmls or have a case class that return the two parts and let the UI decided how to show it

`case class FullCalculationDetails(year1: Html, year2: Html)`

Reply:
You can do that with nonTabCalculationResultsTable. Like this:
nonTabCalculationResultsTable(
    Seq(year1),
    daysInAccountingPeriod,
    associatedCompanies,
    taxableProfit,
    distributions,
    config
  )

Reply:
You dont want the caller (view) to pass in all these details. The helper should take on more responsibility

Reply:
Just had chat with Adam on this, what you have done looks good @PavelVjalicin 

We will proceed with this approach for now